packages/main/src/plugin/kubernetes/context-resource-registry.ts

```diff
@@ -47,9 +47,12 @@ export class ContextResourceRegistry<T> {
     });
   }
 
-  getForResource(resourceName: string): Details<T>[] {
+  getForContextsAndResource(contextNames: string[], resourceName: string): Details<T>[] {
```

Comment:
Weird naming? getFor what? get details ? what contain details ? resource ?

Reply:
As this is a generic class, it is difficult to be more precise